src/frontend/src/tests/lib/derived/exchange.derived.spec.ts

```diff
@@ -0,0 +1,31 @@
+import * as exchangeEnv from '$env/exchange.env';
+import { exchangeInitialized } from '$lib/derived/exchange.derived';
+import { exchangeStore } from '$lib/stores/exchange.store';
+import { get } from 'svelte/store';
+
+describe('exchange.derived', () => {
```

Comment:
Maybe add a test with `EXCHANGE_DISABLED` explicitely set to `undefined` and `false`, just in case?

Reply:
done!